### ISSUE TITLE
Uncaught exception when using `$auth['allow_custom_html'] = true;`

### ISSUE THREAD

**Issue by stueja** [meeting-room-booking-system/mrbs-code#3728]
**Describe the bug**
When `$auth['allow_custom_html'] = true;` is set, the error message appears in `edit_room.php?room=3`

```
Uncaught exception 'TypeError' in /var/www/html/lib/MRBS/Form/Field.php at line 134
MRBS\Form\Field::setControlText(): Argument #1 ($text) must be of type string, null given, called in /var/www/html/edit_room.php on line 248
#0 /var/www/html/edit_room.php(248): MRBS\Form\Field->setControlText()
#1 /var/www/html/edit_room.php(324): MRBS\get_fieldset_general()
#2 {main}

MRBS GET: Array
(
    [room] => 3
)
MRBS POST: Array
(
)
MRBS SESSION: Array
(
    [last_page] => /edit_room.php?room=3
    [this_page] => /edit_room.php?room=3
    [csrf_token] => c192dbb8397c9f736b5a43ce4674b7d76a2f8789784a7bcf45b5c01fbcdef186
    [user] => MRBS\User Object
        (
            [username] => user
            [display_name] => User Name
            [email] => user.name@email.de
            [level] => 2
            [data:protected] => Array
                (
                    [id] => 1
                    [password_hash] => some hash
                    [timestamp] => 2024-02-19 20:25:47
                    [last_login] => 1724007464
                    [reset_key_hash] =>
                    [reset_key_expiry] => 0
                )

        )

)

#0 MRBS\generate_backtrace() called at [/var/www/html/functions_error.inc:226]
#1 MRBS\output_error() called at [/var/www/html/functions_error.inc:280]
#2 MRBS\output_exception_error(object) called at [/var/www/html/functions_error.inc:288]
#3 MRBS\exception_handler(object)
Oh je! Leider ist in MRBS ein schwerwiegender Fehler aufgetreten. Bitte konsultieren Sie Ihren Systemadministrator.
```

**To Reproduce**
1. in `config.inc.php`, add `$auth['allow_custom_html'] = true;`
2. Click on Rooms
3. Click on a room in the list

**Expected behavior**
The room edit page is shown.

**Screenshots**

**Browser details (please complete the following information):**
 - OS: Arch Linux 6.10.0
 - Browser Firefox
 - Version 129.0.1

**Server details (please complete the following information):**
 - MRBS version 1.11.5
 - Web server: nginx/1.22.1
 - OS: Linux 0a4bb6666050 5.15.0-113-generic 123-Ubuntu
 - PHP version: 8.1.20
 - Database used: mariadb 11.4.3

**Additional context**

**Comment by campbell-m** [comment]
Now fixed in [d1f1180](https://github.com/meeting-room-booking-system/mrbs-code/commit/d1f1180623db0e70d072da9c24090f41702e38df).

**Comment by stueja** [comment]
Can't decide whether this was the fastest fix, or the one before... Thanks a great bunch, works perfectly.
